src/libcmis/ws-relatedmultipart.cxx

```diff
@@ -263,7 +263,7 @@ string RelatedMultipart::getContentType( )
         string startType = start->getContentType( );
         size_t pos = startType.find( ";" );
         if ( pos != string::npos )
-            startType = startType.substr( 0, pos );
+            startType.pop_back();
```

Comment:
I'm not sure about this one, the other two are 
X = X.substr( 0, X.length() - 1 )
and are clearly dropping the last character, but in this case pos is from a find of ";"?

Reply:
Thank you for your feedback Caolán , I reverted this specific part.